refactor: improve linter and formatter intros (#142)
biomejs/website: src/content/docs/formatter/index.mdx

import PackageManagerBiomeCommand from "@/components/PackageManagerBiomeCommand.astro";

Biome is an opinionated formatter that [supports multiple languages](/internals/language-support).
It follows a similar [philosophy to Prettier](https://prettier.io/docs/en/option-philosophy.html),
only supporting a few options to avoid debates over styles, turning into debates over Biome options.
It deliberately [resists the urge to add new options](https://github.com/prettier/prettier/issues/40) to prevent [bike-shed discussions](https://en.wikipedia.org/wiki/Law_of_triviality) in teams so they can focus on what really matters instead.

## CLI

The following command checks the formatting of the files in the `src` directory.
It emits text differences if it finds code that is not formatted.

<PackageManagerBiomeCommand command="format ./src" />

If you want to **apply** the new formatting, pass the `--write` option:

<PackageManagerBiomeCommand command="format --write ./src" />

> The command accept a list of files and diretcories.
> Note that if you pass a glob as a parameter, it will be expanded by your shell.
> The result of the expansion depends on your shell and its configuration.
> For example, some shells don't support the recursive glob `**` or the alternation `{}` in the following command:

Reviewer: ```suggestion
The command accepts a list of files and directories.
Note that if you pass a glob as a parameter, your shell will expand it.
The result of the expansion depends on your shell and its configuration.
For example, some shells don't support the recursive glob `**` or the alternation `{}` in the following command:
```

Maybe we should put this paragraph inside a `:::note`, it's something require a bit of attention

Reviewer: Also, we should add a `:::caution` that says that shell expansion of patterns might slow down the command by a lot.

    0,
  ];
```

## Ignoring files

Biome doesn't provide ignore comments that ignore an entire file.
However, you can ignore a file using the Biome configuration file.

You can ignore files for all tools, including the formatter,
using the [`files.ignore` configuration](/guides/how-biome-works#include-and-ignore-explained).
By default, Biome ignores the [protected files](/guides/how-biome-works#protected-files).

> If you want that only the formatter to ignore some files, you can use `linter.ignore`:

Reviewer: ```suggestion
If you want to exclude files from being formatted, you can use `formatter.ignore`:
```
